kalite/distributed/static/js/distributed/exercises.js

```diff
@@ -222,7 +222,12 @@ window.AttemptLogCollection = Backbone.Collection.extend({
     },
 
     url: function() {
-        return "/api/attemptlog/?" + $.param(this.filters, true);
+        if (window.statusModel.get("is_logged_in")) {
+            return "/api/attemptlog/?" + $.param(this.filters, true);
+        }
+        else {
+            window.location = "/securesync/login";
```

Comment:
I know this is already merged but let's minimize hard-coding urls on the front-end.  

Here's a suggestion which is pretty common in the codebase, on the html template put this on the `head` tag:

```javascript
var LOGIN_URL = {% url "login" %};
```

and you can then do this on the .js file:

```javascript
window.location = LOGIN_URL;
```